schemas/ephys/ephys_processing.json

```diff
@@ -0,0 +1,193 @@
+                    "description": "Path to code respository"
+                },
+                "notes": {
+                    "type": "string"
+                },
+                "name": {
+                    "anyOf": [
+                        {
+                            "title": "preprocessing",
+                            "type": "object",
+                            "properties": {
+                                "highpass_filter": {
+                                    "type": "object",
+                                    "properties": {
+                                        "freq_min": {
+                                            "type": "number",
+                                            "title": "Frequency minimum"
+                                        },
+                                        "margin_ms": {
+                                            "type": "number",
+                                            "title": "Margin (ms)",
+                                            "units": "ms"
+                                        }
+                                    },
+                                    "required": ["freq_min", "margin_ms"]
+                                },
+                                "phase_shift": {
+                                    "type": "object",
+                                    "properties": {
+                                        "margin_ms": {
+                                            "type": "number",
+                                            "title": "Margin (ms)",
+                                            "units": "ms"
+                                        }
+                                    },
+                                    "required": ["margin_ms"]
+                                },
+                                "common_refrence": {
+                                    "type": "object",
+                                    "properties": {
+                                        "reference": {
+                                            "type": "string",
+                                            "title": "Reference",
+                                            "enum": ["global"]
+                                        },
+                                        "operator": {
+                                            "type": "string",
+                                            "title": "Operator",
+                                            "enum": ["median"]
+                                        }
+                                    },
+                                    "required": ["reference", "operator"]
+                                }
+                            },
+                            "required": ["highpass_filter", "phase_shift", "common_reference"]
+                        },
+                        {
+                            "title": "align_params_Ex_488_Em_525",
```

Comment:
is this supposed to be in imaging instead of ephys?

Reply:
no, I just haven't deleted yet from ephys... I duplicated the same file into two and was editing one and just had the other as a place holder.